pkg/kubectl/cmd/convert.go

```diff
@@ -230,7 +230,13 @@ func asVersionedObject(infos []*resource.Info, forceList bool, version schema.Gr
 		object = objects[0]
 	} else {
 		object = &api.List{Items: objects}
-		converted, err := tryConvert(scheme.Scheme, object, version, scheme.Registry.GroupOrDie(api.GroupName).GroupVersion)
+		targetVersions := []schema.GroupVersion{}
```

Comment:
@deads2k is there an upstream PR for this? 00000 is not much helpful for the next rebaser.